docs/CoC.md

```diff
@@ -63,7 +63,7 @@ If a user channel is inactive for 30+ days, a staff member will send a message a
 - **#announcements** - Used for announcements related to MMD, Minecraft, the community and partners, along with important world events. 
 - **#artist-talk** - Used to discuss art you've made, or wanted to share. Keep the chat SFW!
 - **#bot-spam** - Chat is for using commands on the official MMD bots.
-- **#community-announcements** - Used by the community to share announcements for their own projects. Responses and discussion about said announcements should be done in a separate channel, reaction emoji are fine though. 
+- **#community-announcements** - Used by the community to share announcements for their own projects. We request that you link to the project overview page rather than a direct download. Responses and discussion about announcements should be done in a separate channel, however adding reaction emoji is fine.
```

Comment:
Shouldn't there be a comma after `however` here?

Reply:
Agree, needs a comma after however. Otherwise fine.